tests/trainer/test_dataloaders.py

```diff
@@ -96,76 +116,16 @@ def test_dataloader_config_errors_init(tmpdir, dataloader_options):
         Trainer(default_root_dir=tmpdir, max_epochs=1, **dataloader_options)
 
 
-def test_multiple_val_dataloader(tmpdir):
```

Comment:
it's extremely hard to review, because it's not clear where things are being moved.
Please consider only converting EvalModelTemplate and do the moving of tests in another PR.